Cargo.toml

```diff
@@ -32,17 +37,8 @@ version = "2.1"
 
 [dev-dependencies]
 bencher = "0.1.5"
-
-# Cargo doesn't let you publish crates with circular dev dependencies,
-# so in order to publish new versions you'll need to temporarily comment out
-# this dependency in your local codebase and publish the crate using the
-# --allow-dirty flag.
-#
-# This is a known problem in Cargo - see
-# https://github.com/rust-lang/cargo/issues/4242 for tracking issue.
```

Comment:
Would prefer we left this comment here.

Reply:
I think it has to move to the derive crate, but yes, it still needs to exist,